ComponentKit/Utilities/CKComponentAction.h

```diff
@@ -101,14 +91,6 @@ class CKTypedComponentAction : public CKTypedComponentActionBase {
   
 public:
   CKTypedComponentAction<T...>() noexcept : CKTypedComponentActionBase() {};
-  CKTypedComponentAction<T...>(id target, SEL selector) noexcept : CKTypedComponentActionBase(target, selector)
-  {
-#if DEBUG
-    std::vector<const char *> typeEncodings;
-    CKTypedComponentActionTypeVectorBuild(typeEncodings, CKTypedComponentActionTypelist<T...>{});
-    _CKTypedComponentDebugCheckTargetSelector(target, selector, typeEncodings);
```

Comment:
Looks like we can delete this function:

https://github.com/facebook/componentkit/search?utf8=%E2%9C%93&q=_CKTypedComponentDebugCheckTargetSelector&type=